### PR TITLE
Fix 3.10.0

### PR DESCRIPTION
**Pull request by naqvis5**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
     "@dvsa/mes-config-schema": "^1.1.0",
     "@dvsa/mes-journal-schema": "1.2.0",
     "@dvsa/mes-search-schema": "1.1.1",
-    "@dvsa/mes-test-schema": "3.31.0",
```
Comment (andrewsetterfield) [inline package.json]: um, why has this changed.

**Review by andrewsetterfield — CHANGES_REQUESTED** [review]

File: test/e2e/pages/loginPage.ts [hunk 2]
```diff
@@ -68,20 +69,15 @@ class LoginPage extends Page {
             const signinForOrgAccount = element(by.xpath(`//XCUIElementTypeButton[@name="Sign in"]`));
             return signinForOrgAccount.click();
           })
-          .then((result) => {
-            return browser.sleep(TEST_CONFIG.PAGE_LOAD_WAIT);
-          })
+          // .then((result) => {
```
Comment (andrewsetterfield) [inline test/e2e/pages/loginPage.ts]: Can this commented out code be removed?

File: test/e2e/pages/waitingRoomToCarPage.ts [hunk 3]
```diff
@@ -216,7 +216,7 @@ class WaitingRoomToCarPage extends Page {
 
   selectSafetyAndBalanceQuestions(table, pageTitle) {
     this.openSelectQuestionsOverlay();
-    PageHelper.waitForOverlay('click-block-active');
+    // PageHelper.waitForOverlay('click-block-active');
```
Comment (andrewsetterfield) [inline test/e2e/pages/waitingRoomToCarPage.ts]: Can this commented out code be removed.

File: test/e2e/pages/loginPage.ts [hunk 4]
```diff
@@ -68,20 +69,15 @@ class LoginPage extends Page {
...
+          //   return browser.sleep(TEST_CONFIG.PAGE_LOAD_WAIT);
+          // })
```
Comment (tomcrawleyy) [inline test/e2e/pages/loginPage.ts]: commented out code should be removed

File: test/e2e/pages/waitingRoomToCarPage.ts (hunk 3 above)
Comment (tomcrawleyy) [inline test/e2e/pages/waitingRoomToCarPage.ts]: remove this

File: test/e2e/step-definitions/generic-steps.ts [hunk 5]
```diff
@@ -200,7 +198,7 @@ Then('I should see the {string} page', (pageTitle) => {
 });
 
 Given('I am on the {string} page', (pageTitle) => {
-  PageHelper.waitForOverlay('click-block-active');
+  //PageHelper.waitForOverlay('click-block-active');
```
Comment (tomcrawleyy) [inline test/e2e/step-definitions/generic-steps.ts]: and here

Comment (andrewsetterfield) [inline test/e2e/step-definitions/generic-steps.ts]: Commented out code, can it be removed.

**Review by andrewsetterfield — CHANGES_REQUESTED** [review]

File: test/e2e/step-definitions/generic-steps.ts (hunk 5 above)
Comment (naqvis5) [reply test/e2e/step-definitions/generic-steps.ts]: done

File: test/e2e/pages/waitingRoomToCarPage.ts (hunk 3 above)
Comment (naqvis5) [reply test/e2e/pages/waitingRoomToCarPage.ts]: done

File: test/e2e/pages/loginPage.ts (hunk 2 above)
Comment (naqvis5) [reply test/e2e/pages/loginPage.ts]: done

File: test/e2e/step-definitions/generic-steps.ts (hunk 5 above)
Comment (naqvis5) [reply test/e2e/step-definitions/generic-steps.ts]: done

File: test/e2e/pages/waitingRoomToCarPage.ts (hunk 3 above)
Comment (naqvis5) [reply test/e2e/pages/waitingRoomToCarPage.ts]: done

File: test/e2e/pages/loginPage.ts (hunk 4 above)
Comment (naqvis5) [reply test/e2e/pages/loginPage.ts]: done

File: test/e2e/features/cat-b-Part2/06-debrief.feature [hunk 6]
```diff
@@ -50,36 +48,3 @@ Feature: Debrief including Health Declaration
     And I should see the "Debrief - Miz Kiff" page
     When I end the debrief
     Then I should see the "Test debrief - Miz Kiff" page
-
-  Scenario: For a pass the health declaration shows the correct information and validation is enforced
```
Comment (turnerc1) [inline test/e2e/features/cat-b-Part2/06-debrief.feature]: Why was this whole scenario removed?

File: test/e2e/pages/waitingRoomToCarPage.ts (hunk 3 above)
Comment (tomcrawleyy) [reply test/e2e/pages/waitingRoomToCarPage.ts]: is it?

File: test/e2e/features/cat-b-Part2/06-debrief.feature (hunk 6 above)
Comment (naqvis5) [reply test/e2e/features/cat-b-Part2/06-debrief.feature]: done

File: test/e2e/pages/waitingRoomToCarPage.ts (hunk 3 above)
Comment (naqvis5) [reply test/e2e/pages/waitingRoomToCarPage.ts]: now?

**Review by andrewsetterfield — APPROVED** [review]

**Review by turnerc1 — APPROVED** [review]

**Pull request merged**